src/snapred/backend/dao/state/PixelGroupingParameters.py

```diff
@@ -4,6 +4,7 @@
 
 
 class PixelGroupingParameters(BaseModel):
+    groupID: int
```

Comment:
From the design standpoint, I am concerned that `groupID` is going to be "grouped" in one class with physical parameters: `dmin`/`dmax`, `2Theta`.  @rboston628, I thought you were going to keep those things separate, i.e. create a dictionary with the group ID being the key and the physical parameters being the value.

Reply:
I can see if there's a way to avoid grouping the group ID in with the physical parameters.  Following on of @walshmm 's comments, we don't want to specify the group ID as a value to a key, i.e. `"groupID": 2` instead of just `2`.  I'll look around for another way to include it, that doesn't require yet another DAO.

Reply:
Please also see @walshmm's earlier comment on [Story 1532](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20%28Change%20Management%29#action=com.ibm.team.workitem.viewWorkItem&id=1532): "So there will be a map of (group,data) pairs.."